pyomo/contrib/gdpopt/util.py

```diff
@@ -260,7 +252,7 @@ def is_feasible(model, config):
             config.logger.info('%s: body %s > UB %s' % (
                 constr.name, value(constr.body), value(constr.upper)))
             return False
-    for var in model.component_data_objects(ctype=Var, descend_into=True):
+    for var in model.component_data_objects(ctype = Var, descend_into = True):
```

Comment:
There's a lot of these whitespace changes, so I won't comment on each one, but you may refer here: https://www.python.org/dev/peps/pep-0008/#other-recommendations

Reply:
Maybe there is something wrong with my editor. I'll check that.